config/test.exs

```diff
@@ -1,11 +1,8 @@
-use Mix.Config
+import Config
 
 config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
   http: [port: 4001],
   server: false,
-  secret_key_base: "G0ieeRljoXGzSDPRrYc2q4ADyNHCwxNOkw7YpPNMa+JgP9iGgJKT4K96Bw/Mf/pd",
-  session_key: "test",
-  signing_salt: "qh+vmMHsOqcjKF3TSSIsghwt2go48m2+IQ+kMTOB3BrSysSr7D4a21uAtt4yp4wn",
```

Comment:
This means that when we run `mix test` we need to have more than the `DATABASE_URL` env var?

Reply:
See my response to @remiprev above. I’m open for proposal of a better way to wire our environment variables script…